UNITTESTS/features/cellular/framework/device/cellularcontext/unittest.cmake

```diff
@@ -45,6 +45,16 @@ set(unittest-test-sources
   stubs/Mutex_stub.cpp
 )
 
-# defines
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DMDMRTS=PTC0 -DMDMCTS=PTC1 -DMDMTXD=NC -DMDMRXD=NC -DMBED_CONF_PLATFORM_DEFAULT_SERIAL_BAUD_RATE=115200 -DCELLULAR_DEVICE=myCellularDevice -DDEVICE_SERIAL_FC=1 -DMBED_CONF_CELLULAR_CONTROL_PLANE_OPT=0")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DMDMRTS=PTC0 -DMDMCTS=PTC1 -DMDMTXD=NC -DMDMRXD=NC -DMBED_CONF_PLATFORM_DEFAULT_SERIAL_BAUD_RATE=115200 -DCELLULAR_DEVICE=myCellularDevice -DDEVICE_SERIAL_FC=1 -DMBED_CONF_CELLULAR_CONTROL_PLANE_OPT=0")
+set(unittest-test-flags
+  -DMDMRTS=PTC0
+  -DMDMCTS=PTC1
+  -DMDMTXD=NC
+  -DMDMRXD=NC
+  -DMBED_CONF_PLATFORM_DEFAULT_SERIAL_BAUD_RATE=115200
+  -DCELLULAR_DEVICE=myCellularDevice
```

Comment:
Not directly related to this PR, but `CELLULAR_DEVICE` flag is no longer in use. (Couple of other cmake files has this too).